fixedassoc.py

```diff
@@ -0,0 +1,976 @@
+#!/usr/bin/env python
```

Comment:
This script should go to the `try` directory, and possibly be consolidated with the other script.

Reply:
Whoops put into the tools directoyr

Reply:
Fixed